prow/config/jobs/envoy-1.12.yaml

```diff
@@ -0,0 +1,50 @@
+org: istio
+repo: envoy
+branches: [release-1.12]
+image: envoyproxy/envoy-build-ubuntu:81a93046060dbe5620d5b3aa92632090a9ee4da6
+node_selector:
+  testing: build-pool
+
+jobs:
+  - name: test-asan
+    types: [presubmit]
+    env:
+      - name: BAZEL_BUILD_EXTRA_OPTIONS
+        value: "--local_ram_resources=131072 --local_cpu_resources=42 --test_env=ENVOY_IP_TEST_VERSIONS=v4only --flaky_test_attempts=9"
+      - name: ENVOY_SRCDIR
+        value: "/home/prow/go/src/istio.io/envoy"
+      - name: FILTER_WORKSPACE_SET
+        value: "false"
+    command: [./ci/do_ci.sh, bazel.asan]
+    timeout: 4h0m0s
+
+  - name: test-tsan
+    types: [presubmit]
+    env:
+      - name: BAZEL_BUILD_EXTRA_OPTIONS
+        value: "--local_ram_resources=131072 --local_cpu_resources=42 --test_env=ENVOY_IP_TEST_VERSIONS=v4only --flaky_test_attempts=9"
+      - name: ENVOY_SRCDIR
+        value: "/home/prow/go/src/istio.io/envoy"
+      - name: FILTER_WORKSPACE_SET
+        value: "false"
+    command: [./ci/do_ci.sh, bazel.tsan]
+
+  - name: test-release
+    types: [presubmit]
+    env:
+      - name: BAZEL_BUILD_EXTRA_OPTIONS
+        value: "--local_ram_resources=131072 --local_cpu_resources=42 --test_env=ENVOY_IP_TEST_VERSIONS=v4only --flaky_test_attempts=9"
+      - name: ENVOY_SRCDIR
+        value: "/home/prow/go/src/istio.io/envoy"
+    command: [./ci/do_ci.sh, bazel.release]
+
```

Comment:
Removed the auto-update job

Reply:
I had thought we wanted to keep the autoupdate job so RMs didn't have to know to pull newer releases and we said that any envoy PR in the release stream was good. Maybe we could leave the job and remove the auto-merge so the RM knows a change is there, but has the ability to say whether it merges or not.

Reply:
This is for the envoy repo. We only need this for istio-private, not for istio/istio.

Reply:
We don't want it to update istio(-private)/proxy with the istio(-private)/envoy repository. Hopefully that makes it clear.